docs/projects/blank/overview.md

```diff
@@ -1,3 +1,21 @@
-# Blank Template project
+# Blank Starter Project
 
-// Information about the blank template project
+The Blank starter project is a great starting point for working on the SpatialOS GDK for Unity if you want a clean slate.
```

Comment:
Is it called "Blank Starter Project" (proper noun) or "Blank starter project"?

The "FPS Starter Project" is a proper noun in all the docs (except, I see where it isn't - I didn't get to look at the docs in the project directory!)

It's important to be consistent as this aids user understanding - they aren't puzzling over whether we mean the same thing when we write the name of this.

In addition, if it's a proper noun, keep capital initials in the title. If it's not, remove them as our title standard is newspaper titles style (like a sentence but without a full-stop).

Reply:
Two "fors" in the sentence make it hard to read. "Clean slate" is an English-language colloquialism which may not mean anything to non-US and non-UK users.

Reply:
(See above for edited version of this doc.)

Reply:
added full changes from above